ansible/roles/preseed/templates/preseed_debian-stretch.cfg.j2

```diff
@@ -60,8 +60,19 @@ d-i partman-md/device_remove_md boolean true
 d-i partman-lvm/confirm boolean true
 d-i partman-lvm/confirm_nooverwrite boolean true
 
+{% if (hostvars[hostname].install_cooked.efi | default(false)) %}
+d-i partman-efi/non_efi_system boolean true
```

Comment:
I approved a bit fast.  `partman-efi/non_efi_system` is set to true when installing on an EFI system?  O_o